chord_metadata_service/chord/workflows/metadata.py

```diff
@@ -147,6 +149,30 @@ def json_file_output(id_: str, output_name: Optional[str] = None):
                 }
             ]
         },
+        WORKFLOW_READDOC: {
+            "name": "Document",
+            "description": "XThis workflow will send document files to the specified ingestion service.",
```

Comment:
description doesn't really make sense - should be maybe something like

> This workflow ingests into DRS documents which have been already listed as experiment results.`

Reply:
Perfect!!